### ISSUE TITLE
Player Color for PC Actor in PF2E: All Same Color

### ISSUE THREAD

**Issue by Armandeus66** [Dorako/pf2e-dorako-ui#344]
Thank you for considering this! It's awesome!

All of my players (with different colors) have actor sheets with the color from only one of the players, but I appreciate that this is a work in progress.

**Comment by Dorako** [comment]
Yeah, looks like I'll need a better way of determining who's the "real owner" of an actor

**Comment by Armandeus66** [comment]
Thanks for updating and working on this, but I still seem to have the same issue.

**Comment by Dorako** [comment]
I have done nothing to change it, so that checks out. I'll be sure to write it in the patch notes.

**Comment by Armandeus66** [comment]
Sorry, I saw an update, and jumped to conclusions. My bad.

**Comment by Dorako** [comment]
Is it fixed now in 3.7.4?

**Comment by Armandeus66** [comment]
Thanks! It works! Awesome!

**Comment by Armandeus66** [comment]
With the latest update, now all the actors use one player's color and not the other. Both players have observer status on each others' sheets.

**Comment by Dorako** [comment]
So it worked for you in 3.7.4 but not in 3.7.5? The only thing I did related to sheet colors, was introducing an early termination check for the case where there's no actor (which obviously should never be the case for character sheets).

**Comment by Armandeus66** [comment]
I think it worked in 3.7.4. I remember checking and being impressed that it did. Maybe I did something wrong.
You can reproduce my issue by creating 4 PC actors and 2 players. Give Player A 2 characters and Player B 2 as well. Give each player observer permission on the other's 2 actors. "Player A" should come before "Player B" in the permission list, and if the same issue appears, Player B's owned actors should be Player A's color.
Thanks for working on this!

**Comment by Dorako** [comment]
Is your issue fixed with 3.7.6?

**Comment by Armandeus66** [comment]
Yes, thank you!
